pages/overview.js

```diff
@@ -1,25 +1,48 @@
 import styled from "styled-components";
+import { useRouter } from "next/router";
+
+export default function Overview({ bankAccount, spendingValues }) {
+  const router = useRouter();
+  let newKontostand = 0;
+  let allSpendingsTogether = 0;
+
+  if (spendingValues.length > 0) {
+    spendingValues.map((spendingValues) => {
+      allSpendingsTogether = allSpendingsTogether + parseFloat(spendingValues.spending);
+    });
+  }
+
+  newKontostand = bankAccount.kontostand - allSpendingsTogether;
 
-export default function Overview({ bankAccount }) {
   return (
     <main>
       <StyledHeader>
         <StyledHeadline>Willkommen {bankAccount.name}</StyledHeadline>
       </StyledHeader>
       <StyledParagraph>dein Kontostand beträgt zur Zeit:</StyledParagraph>
-      <StyledDiv>{bankAccount.kontostand}€</StyledDiv>
+      <StyledDiv>{newKontostand}€</StyledDiv>
+      <MinusBtn onClick={() => router.push("/moneyspending")}>-</MinusBtn>
+      <StyledHeadlineTwo>Zahlungsverlauf</StyledHeadlineTwo>
+      <StyledTable>
+        {spendingValues.map((spendingValues) => (
+          <StyledTr>
+            <LeftTd>{spendingValues.spendFor}</LeftTd>
+            <RightTd>-{spendingValues.spending}€</RightTd>
+          </StyledTr>
+        ))}
+      </StyledTable>
```

Comment:
A table is a bit of over-engineered. Why don't you use the more common and modern way with `flex` or `grid`